src/Model/Order/Product/AssocVxc.php

```diff
@@ -51,7 +51,7 @@ public function __construct(string $name, ConnectableProduct $aEnd, ConnectableP
      */
     public function getName(): string
```

Comment:
probably should change this method name as well to make it consistent

Reply:
agreed, did that